airflow/providers/mysql/transfers/s3_to_mysql.py

```diff
@@ -24,7 +24,7 @@
 from airflow.utils.decorators import apply_defaults
 
 
-class S3ToMySqlTransferOperator(BaseOperator):
+class S3ToMySqlOperator(BaseOperator):
     """
     Loads a file from S3 into a MySQL table.
```

Comment:
Can I ask what's the requirement S3 file format for this Operator? json? csv? parquet and etc.

Reply:
> The LOAD DATA statement reads rows from a text file into a table at a very high speed.

You can define sth like:
```
    [{FIELDS | COLUMNS}
        [TERMINATED BY 'string']
        [[OPTIONALLY] ENCLOSED BY 'char']
        [ESCAPED BY 'char']
    ]
    [LINES
        [STARTING BY 'string']
        [TERMINATED BY 'string']
    ]
```
for csv files.

Reply:
Thanks @feluelle, this helped a lot!

Reply:
One more question! Does this work with parquet files? After looking into the docs, I don't think it does. Right?

Reply:
No I don't think it works directly. You would have to convert parquet to csv and load csv to mysql.